javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,13 @@
+//Goodboy-Oldboy
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+var dogYear = dogYearFuture - dogYearOfBirth;
+{
+let shouldShowResultInDogYears = false;
+console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
+}
+{
+let shouldShowResultInDogYears = true;
+var dogYear = dogYear * 7;
+console.log('Your dog will be ' + dogYear + ' years old in ' + dogYearFuture);
+}
```

Comment:
I haven't really seen this syntax before. 
Good thinking on how to split the code into sections, but you shouldn't do it like this. 

If you want to test both cases you could duplicate your code like this:
```javascript
let shouldShowResultInDogYears = false;

if (shouldShowResultInDogYears) {
  console.log('Your dog will be ' + dogYear + ' years old in ' + dogYearFuture);
} else {
  console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
}

shouldShowResultInDogYears = true;

if (shouldShowResultInDogYears) {
  console.log('Your dog will be ' + dogYear + ' years old in ' + dogYearFuture);
} else {
  console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
}
```

Reply:
In the class thread- someone mentioned that we are not supposed to use if else, so I used this code in bracket method. But now I have updated according to your feedback

Reply:
Oh ok. I didn't know that. 
You're gonna end up using a lot of if/else-statements in the future so you might as well start now :)